core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -533,6 +546,19 @@ func (ec *EthConfirmer) batchFetchReceipts(ctx context.Context, attempts []EthTx
 			promNumSuccessfulTxs.WithLabelValues(ec.chainID.String()).Add(1)
 		}
 
+		// This is only recording forwarded tx that were mined and have a status.
+		// Counters are prune to being in-accurate due to re-orgs.
+		if ec.config.EvmUseForwarders() {
+			meta, err := attempt.EthTx.GetMeta()
+			if err != nil {
+				continue
+			}
+			if meta.FwdrDestAddress != nil {
+				// promFwdTxCount takes two labels, chainId and a boolean of whether a tx was successful or not.
+				promFwdTxCount.WithLabelValues(ec.chainID.String(), fmt.Sprintf("%t", receipt.Status != 0)).Add(1)
```

Comment:
nit/ alternatively:
```suggestion
				promFwdTxCount.WithLabelValues(ec.chainID.String(), strconv.FormatBool(receipt.Status != 0)).Add(1)
```
I guess it's actually one character longer this way :shrug:

Reply:
Yeah I guess that's more readable. Commited